macros/src/deps.rs

```diff
@@ -79,26 +79,24 @@ impl Dependencies {
 
 impl ToTokens for Dependencies {
     fn to_tokens(&self, tokens: &mut TokenStream) {
-        let crate_rename = &self.crate_rename;
         let lines = self.dependencies.iter();
 
-        tokens.extend(quote![{
-            use #crate_rename::typelist::TypeList;
-            ()#(#lines)*
-        }]);
+        tokens.extend(quote![
+            #(#lines;)*
```

Comment:
I love that there is no longer a need for the unit value before expanding `lines`